example/conf/a.conf

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
All of `example/conf` and `imgs/` seem like they should be part of the test suite as well.